tests/main/interfaces-daemon-notify/task.yaml

```diff
@@ -29,14 +29,14 @@ execute: |
 
     denials_before="$(get_journalctl_log -u snap.test-snapd-daemon-notify.notify.service | grep -c 'Permission denied' || true)"
     echo "Then after we restart the servive there is not denials"
+    snap start test-snapd-daemon-notify.notify
     for _ in $(seq 10); do
-        if snap start test-snapd-daemon-notify.notify; then
+        denials_after="$(get_journalctl_log -u snap.test-snapd-daemon-notify.notify.service | grep -c 'Permission denied' || true)"
+        if [ "$denials_before" -eq "$denials_after" ]; then
```

Comment:
If the problem is logs appearing after a longer/unexpected delay, then this check should be reversed to `"$denials_before" -ne "$denials_after"`. Otherwise, it only confirms that there are as many denials as before, which can be caused by the sheer lack of denials (and the functionality working correctly), or that denials have not been logged yet.

Reply:
@bboozzoo thanks for the review, the changes are already pushed